gsa/src/web/pages/performance/performancepage.js

```diff
@@ -162,7 +162,9 @@ const Selector = withClickHandler()(styled.span`
   }}
 `);
 
-const SLAVE_SCANNER_FILTER = Filter.fromString('type=' + GMP_SCANNER_TYPE);
+const SLAVE_SCANNER_FILTER = Filter.fromString(
```

Comment:
Would be nice if we could rename this variable in another PR. Something like SENSOR_SCANNERS_FILTER